setup.py

```diff
@@ -7,7 +7,7 @@ def convert_markdown_to_rst(file):
 
 
 setup(name='gitlabform',
-      version='1.5.0',
+      version='1.5.1',
```

Comment:
This PR introduces 2 new features, so definitely not patch but minor version should be update. I suggest `1.6.0` then.

Reply:
...but it would be even better to not update version in the PR. We will update it directly in master during release of new version.